tests/test_effect_harmonization.py

```diff
@@ -42,6 +42,6 @@ def test_reverse_complement(call_get_reverse_complement: DataFrame) -> None:
     """Test reverse complement."""
     assert (
         call_get_reverse_complement.filter(
-            f.col("reverseComp") != f.col("revcomp_allele")
+            f.col("reverseComp") != f.col("revcompAllele")
```

Comment:
This test is failing because `revcompAllele` is not a column of the df. The effect_harmonization module (where `get_reverse_complement` is defined) is designed in snake case. We would need to change this part of the code https://github.com/opentargets/genetics_etl_python/blob/ab3d4daa3e2e8384ea69a3bef2965df427a51515/src/etl/gwas_ingest/effect_harmonization.py#L60

```suggestion
            f.col("reverseComp") != f.col("revcomp_allele")
```

Reply:
sorry I forgot to push this change